README.md

```diff
@@ -33,6 +33,12 @@ auth:
     scope: openid profile email offline_access
     # Optional id_token claim that will be used for username
     usernameClaim: preferred_username
+    # Optional switch to alternative login method, using access token as password.
+    # The username must be the same as the one used to acquire the access token,
+    # the password must be a base64 encoded access token. The e-mail address will
```

Comment:
I think it is easier in most cases. The OIDC server sends the access token base64 encoded anyway, and we have to send it base64 encoded to the token exchange operation anyway (so we don't decode it, but simply pass it on). The other thing is that we avoid problems with line breaks in formatted JSON if the token is passed in via the npm cli. I would also avoid to allow both representations.

Reply:
Ah, you're talking abound base64 encoding in the JWT? I don't think that it is correct to call it a base64-encoded token, because there are three base64-encoded parts connected by dots.

I propose to change it to
```suggestion
    # the password must be an OIDC access token. The e-mail address will
```

Reply:
You are right the comment is confusing. I incorporated your change with the last update.